src/styles/Navbar.css

```diff
@@ -47,22 +53,33 @@
 .home {
   font-weight: 600;
   margin: auto;
-  padding-right: 30px;
 }
 
 .shop_icon {
   width: 10px;
   height: 10px;
 }
 
-@media (max-width: 1024px) and (min-width: 769px) {
+.nav-buttons {
+  display: flex;
+  gap: 5px;
+}
+
+@media (max-width: 1024px) {
   .navbar-brand {
     font-size: 1.7rem;
   }
   .home {
     margin: auto;
     padding: 8px;
   }
+  .navbar-nav {
+    padding-left: 0px;
+  }
+  .nav-buttons {
+    flex-direction: column;
+    margin: auto;
```

Comment:
In the mobile view, there's too much vertical gap. It would be great if this is changed.
![image](https://github.com/IAmTamal/Milan/assets/72851613/1be75738-2153-428a-95b2-94d669eb959d)